netmiko/nokia/nokia_sros.py

```diff
@@ -299,11 +338,12 @@ def remote_file_size(self, remote_cmd=None, remote_file=None):
             remote_cmd = self._file_cmd_prefix() + "file dir {}/{}".format(
                 self.file_system, remote_file
             )
-        remote_out = self.ssh_ctl_chan.send_command(remote_cmd)
+        remote_out = self.ssh_ctl_chan._send_command_str(remote_cmd)
 
         if "File Not Found" in remote_out:
             raise IOError("Unable to find file on remote system")
 
+        assert remote_file is not None
```

Comment:
I would just put an `else` clause up in the self.direction check where if self.direction is neither "put" nor "get" you raise an exception, that should ensure that remote_file is always set to a string and mypy should be able to work it out from that (hopefully).

Reply:
Yes, mypy is able to process it :)